packages/tinacms/src/media.ts

```diff
@@ -0,0 +1,111 @@
+import { Subscribable } from '@tinacms/core'
+
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+/**
+ * Media represents a single media object (file + metadata) in a media store
+ */
+export interface Media {
+  reference: string // internal reference used by the store
+  src: string // filepath or URI to the media object
+  previewSrc: string // filepath or URI to a preview of the media (will this always be a URI?)
+}
```

Comment:
What about other metadata associated to a media? Images should have an alt and title for instance.

Reply:
Out of scope for this PR. I'll update the description.

Metadata is dependent upon a lot of factors. The file format, storage provider, and usage in the site all effect things. It's far too early to make some kind of opinionated decision about this. Right now, we're leaning towards making it depending on the format. For example, using exif for images, metadata tags for svgs, and god knows what for PDFs. All this is just way more than we want to consider for this initial PR.

Reply:
(I was simply suggesting a media should be accessible by default, `alt` and `title`, are standards in CommonMark and in HTMLsince 1993.)